### PR TITLE
Docker: add openh264 and autopep8 libraries

### PR DESCRIPTION
**Pull request by michael4338**

### CODE REVIEW

**Comment by anchans** [comment]
this installs the shared object files to /usr/local/lib which is not in the LD_LIBRARY_PATH, please add it in apollo_base.sh

File: docker/build/installers/install_openh264.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Prepare
+PACKAGE="v2.0.0.tar.gz"
+OPEN_H264="openh264-2.0.0"
+wget https://github.com/cisco/openh264/archive/${PACKAGE}
+tar zxf ${PACKAGE}
+
+# Build and install.
+pushd ${OPEN_H264}
+  make
+  make install
```
Comment (Capri2014) [inline docker/build/installers/install_openh264.sh]: default path not in `LD_LIBRARY_PATH`, suggest add ` export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH` in `apollo_base.sh`, or try install this one under `/usr/local/openh264` or `/usr/local/apollo/openh264` as the other libs do.

**Comment by hugomatic** [comment]
It looks like `make clean` can be invoked after make install (it won't remove the installed files).

File: docker/build/installers/install_openh264.sh (hunk 1 above)
Comment (michael4338) [reply docker/build/installers/install_openh264.sh]: thinking about adding it in next change, but okay

**Comment by Capri2014** [comment]
will merge in and verify

**Pull request merged**